pint/models/timing_model.py

```diff
@@ -162,6 +163,22 @@ def components(self,):
                 comps[cp.__class__.__name__] = cp
         return comps
 
+    @property
+    def categories(self):
+        categories = []
+        if six.PY2:
+            type_list = super(TimingModel, self).__getattribute__('component_types')
```

Comment:
is __getattribute__ really needed here? It's *extremely* arcane. Shouldn't there be a better interface to getting the list of component types?